virtualizarr/manifests/array_api.py

```diff
@@ -75,7 +76,9 @@ def concatenate(
         axis = axis % first_arr.ndim
 
     arr_shapes = [arr.shape for arr in arrays]
+    arr_chunks = [arr.chunks for arr in arrays]
     check_same_shapes_except_on_concat_axis(arr_shapes, axis)
+    check_no_partial_chunks_on_concat_axis(arr_shapes, arr_chunks, axis)
```

Comment:
Are we also missing a `check_same_chunks` function call here? 😬 Or does it happen inside `check_combinable_zarr_arrays`?

Reply:
It happens inside `check_combinable_zarr_arrays`. I guess that `check_combinable_zarr_arrays` does all the checks that do not depend on the `axis` of concatenation.